src/Icons/FinishedIcon.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { CheckCircleIcon } from '@patternfly/react-icons';
+import './Icons.scss';
+
+const FinishedIcon = () => {
```

Comment:
can rewrite as 👇 for cleanliness
`const FinishedIcon = () => <CheckCircleIcon className="ins-c-dashboard__finished-icon"/>;`